benchmarks/Benchmarks.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <CodeAnalysisRuleSet>$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'OpenTelemetry.sln'))/build/OpenTelemetry.prod.loose.ruleset</CodeAnalysisRuleSet>
```

Comment:
The idea is to drop the `loose` ruleset altogether? If not, sounds like Benchmark could make use of it.